packages/contracts/contracts/interfaces/ISemaphoreVoting.sol

```diff
@@ -0,0 +1,77 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+/// @title SemaphoreVoting contract interface.
+interface ISemaphoreVoting {
+    error SemaphoreVoting__CallerIsNotThePollCoordinator();
+    error SemaphoreVoting__MerkleTreeDepthIsNotSupported();
+    error SemaphoreVoting__PollHasAlreadyBeenStarted();
+    error SemaphoreVoting__PollIsNotOngoing();
+    error SemaphoreVoting__YouAreUsingTheSameNullifierTwice();
+
+    enum PollState {
+        Created,
+        Ongoing,
+        Ended
+    }
+
+    struct Poll {
+        address coordinator;
+        PollState state;
+        mapping(uint256 => bool) nullifiers;
```

Comment:
We can remove this, and the `groupId` parameter as well.

Reply:
you mean nullifiers?

Reply:
Yess, they are already used on `Semaphore.sol`.